pkgs/desktops/elementary/desktop/elementary-gsettings-schemas/default.nix

```diff
@@ -0,0 +1,43 @@
+{ stdenv, runCommand, gnome3, elementary-default-settings, nixos-artwork, glib, gala, epiphany, elementary-settings-daemon, gtk3, plank
+, extraGSettingsOverrides ? ""
+, extraGSettingsOverridePackages ? []
+}:
+
+let
+
+  gsettingsOverridePackages = [
+    gala
+    epiphany
+    elementary-settings-daemon
+    gnome3.mutter
+    gtk3
+    plank
+  ] ++ extraGSettingsOverridePackages;
+
+in
+
+with stdenv.lib;
+
+runCommand "elementary-gsettings-desktop-schemas" {}
```

Comment:
It would probably be nice in the future if we didn't have to duplicate this everywhere as well.

Reply:
What we really need is a proper dconf module with db generation https://github.com/NixOS/nixpkgs/issues/42053#issuecomment-426494778

Reply:
Maybe move that into its own separate issue so we can add it to https://github.com/NixOS/nixpkgs/projects/12.

It's also something I'd like to work on.

Reply:
Will write an issue.

Reply:
Could you add a todo comment linking the issue here?

Reply:
Sure :+1: